swarm_copy/tools/bluenaas_memodel_getall.py

```diff
@@ -55,7 +57,7 @@ async def arun(self) -> PaginatedResponseUnionMEModelResponseSynaptomeModelRespo
         response = await self.metadata.httpx_client.get(
             url=f"{self.metadata.bluenaas_url}/neuron-model/{self.metadata.vlab_id}/{self.metadata.project_id}/me-models",
             params={
-                "simulation_type": self.input_schema.model_type,
```

Comment:
This is an ME model response, not a simulation one. `simulation_type` variable name is not very suitable in this case.

Reply:
I changed it to memodel_type, and at the other places as well.